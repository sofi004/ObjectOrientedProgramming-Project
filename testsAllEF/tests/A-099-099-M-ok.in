4
5
1;1:3;1
7
1
1;1:3;1
5
1;1:5;1
7
1
1;1:5;1
5
1;1:40;1
7
1
1;1:40;1
0
0
