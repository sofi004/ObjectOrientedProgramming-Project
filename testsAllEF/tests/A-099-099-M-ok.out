Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1;1|15
2;1|25
3;1|10
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|
2;1|
3;1|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1;1|
2;1|
3;1|
4;1|23
5;1|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|
2;1|
3;1|
4;1|
5;1|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1;1|
2;1|
3;1|
4;1|
5;1|
6;1|
7;1|
8;1|
9;1|
10;1|
11;1|
12;1|
13;1|
14;1|
15;1|
16;1|
17;1|
18;1|
19;1|
20;1|
21;1|
22;1|
23;1|
24;1|
25;1|
26;1|
27;1|
28;1|
29;1|
30;1|
31;1|
32;1|
33;1|
34;1|
35;1|
36;1|
37;1|
38;1|
39;1|
40;1|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|
2;1|
3;1|
4;1|
5;1|
6;1|
7;1|
8;1|
9;1|
10;1|
11;1|
12;1|
13;1|
14;1|
15;1|
16;1|
17;1|
18;1|
19;1|
20;1|
21;1|
22;1|
23;1|
24;1|
25;1|
26;1|
27;1|
28;1|
29;1|
30;1|
31;1|
32;1|
33;1|
34;1|
35;1|
36;1|
37;1|
38;1|
39;1|
40;1|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 